sources/packages/web/src/components/common/OfferingSummary.vue

```diff
@@ -0,0 +1,249 @@
+        ><font-awesome-icon :icon="['fas', 'search']"
+      /></v-btn>
+      <v-btn
+        v-if="isInstitutionUser"
+        class="ml-2"
+        @click="goToAddNewOffering()"
+        outlined
+        color="#2965C5"
+      >
+        <v-icon size="25" left>
+          mdi-open-in-new
+        </v-icon>
+        Add Study Period
+      </v-btn>
+    </div>
+  </div>
+  <DataTable
+    :value="offeringsAndCount.offeringSummary"
+    :lazy="true"
+    :paginator="true"
+    :rows="DEFAULT_PAGE_LIMIT"
+    :rowsPerPageOptions="PAGINATION_LIST"
+    :totalRecords="offeringsAndCount.totalOfferings"
+    @page="paginationAndSortEvent($event)"
+    @sort="paginationAndSortEvent($event)"
+    :loading="loading"
+  >
+    <template #empty>
+      <p class="text-center font-weight-bold">No records found.</p>
+    </template>
+    <Column
+      :field="OfferingSummaryFields.OfferingName"
+      header="Name"
+      :sortable="true"
+    ></Column>
+    <Column :field="OfferingSummaryFields.StudyDates" header="Study Dates">
+      <template #body="slotProps">
+        {{ formatDate(slotProps.data.studyStartDate, DATE_FORMAT) }} -
+        {{ formatDate(slotProps.data.studyEndDate, DATE_FORMAT) }}
+      </template></Column
+    >
+    <Column :field="OfferingSummaryFields.OfferingIntensity" header="Type"
+      ><template #body="slotProps">
+        <span>{{ slotProps.data.offeringIntensity }} </span>
+      </template>
+    </Column>
+    <Column
+      :field="OfferingSummaryFields.OfferingDelivered"
+      header="Study Delivery"
+    ></Column>
+    <Column>
+      <template #body="slotProps">
+        <v-btn
+          outlined
+          @click="offeringButtonAction(slotProps.data.id)"
+          color="#2965C5"
+        >
+          {{ offeringActionLabel }}
+        </v-btn>
+      </template>
+    </Column>
+  </DataTable>
+</template>
+
+<script lang="ts">
+import { useRouter } from "vue-router";
+import { onMounted, ref, computed } from "vue";
+import {
+  InstitutionRoutesConst,
+  AESTRoutesConst,
+} from "@/constants/routes/RouteConstants";
+import { EducationProgramOfferingService } from "@/services/EducationProgramOfferingService";
+import {
+  PaginatedOffering,
+  ClientIdType,
+  OfferingSummaryFields,
+  DEFAULT_PAGE_LIMIT,
+  DEFAULT_PAGE_NUMBER,
+  DataTableSortOrder,
+  PAGINATION_LIST,
+} from "@/types";
+import { useFormatters } from "@/composables";
+
+export default {
+  props: {
+    programId: {
+      type: Number,
+      required: true,
+    },
+    locationId: {
+      type: Number,
+      required: true,
+    },
+    clientType: {
+      type: String,
+      required: true,
+    },
+  },
+  setup(props: any) {
+    const router = useRouter();
+    const loading = ref(false);
+    const searchBox = ref("");
+    const currentPage = ref();
+    const currentPageLimit = ref();
+    const { formatDate } = useFormatters();
+    const DATE_FORMAT = "MMMM, D YYYY";
```

Comment:
We already this format defined in the _useFormatters_ on the method _dateOnlyLongString_. Can the conde be changed to use it, please?

Reply:
removed. as it no longer needed